scripts/release.js

```diff
@@ -0,0 +1,46 @@
+const shell = require('shelljs');
+const path = require('path');
+const fs = require('fs');
+
+const RELEASE_BRANCH = '1.0.x';
+const cwd = path.resolve('.');
+const pkgPath = path.join(cwd, 'package.json');
+const bowerPath = path.join(cwd, 'bower.json');
+
+function main() {
+	shell.exec(`git checkout ${RELEASE_BRANCH}`);
+	shell.exec(`git stash`);
```

Comment:
Stash before checkout is a good point. To use `stash pop` we need more "smart" script to persist current branch, and track if any changes were stashed